framework/src/auxkernels/AuxKernel.C

```diff
@@ -54,6 +54,11 @@ AuxKernelTempl<ComputeValueType>::validParams()
                         "displacements are provided in the Mesh block "
                         "the undisplaced mesh will still be used.");
   params.addParamNamesToGroup("use_displaced_mesh", "Advanced");
+  params.addParam<bool>("allow_multiple_element_sides",
+                        false,
+                        "Whether to allow an element with multiple sides "
+                        "on the specified boundary to evaluate based on "
+                        "the value on a single side");
```

Comment:
The hardest part: definitely naming this thing. I do know that the parameter text should be more detailed and along the lines of:

> Whether or not to check for repeated elements in the case of a boundary restricted, elemental aux variable. Setting this to false will allow the contribution to a single element's elemental value(s) from multiple boundary sides on the same element (example: when the restricted boundary exists on two or more sides of an element, like at a corner of the mesh).

Parameter name ideas:

- `check_boundary_restricted`
- `check_multiple_boundary_restricted`

Reply:
I like your name ideas better :)

Doc string and error message updated!